### ISSUE TITLE
[FR] Type casting can be used: new pattern

### ISSUE THREAD

**Issue by orklah** [kalessil/phpinspectionsea#1421]
Hi,

I found out today that we had some code like that in our project:
```php
intval($var, 10);
```

According to https://www.php.net/manual/en/function.intval.php, the default value of intval is already 10. Weirdly, this is not the case in PHPStorm stubs(default is null in it) so the second parameter is not reported as unnecessary by PHP EA.

So it could be detected as the same as 

```php
(int)$var;
```

Thanks

**Comment by orklah** [comment]
I just made a PR to the stubs for this. I'll let you judge if this should also be handled by the type casting inspection.

**Comment by kalessil** [comment]
Implemented!

**Comment by orklah** [comment]
Thanks!
